llvm/docs/AMDGPUUsage.rst

```diff
@@ -1755,6 +1755,56 @@ As part of the AMDGPU MC layer, AMDGPU provides the following target specific
 
      =================== ================= ========================================================
 
+Function Resource Usage
+-----------------------
+
+The function resource information (e.g., number of VGPRs) required depends on
+all of its callees' function resources. The expression to denote these
+resources should, therefore, be described as the propagative of its callees'
+equivalent expressions. Said expressions are generated and emitted (as symbols)
```

Comment:
"generated and emitted" could probably be just "emitted", and the parenthesis are not needed around "as symbols":

```
Said expressions are emitted as symbols by the compiler ...
```